CHANGELOG.rst

```diff
@@ -13,6 +13,8 @@ Changes:
 
 - Don't overwrite ``__name__`` with ``__qualname__`` for ``attr.s(slots=True)`` classes.
   `#99 <https://github.com/hynek/attrs/issues/99>`_
+- Attributes can now have metadata dictionaries.
```

Comment:
I feel like you’re underselling your work here!

How about something like:

> Attributes now can have user-defined attribute metadata which greatly improves `attrs`’s extensibility.

?